src/openms/source/CONCEPT/Constants.cpp

```diff
@@ -155,9 +155,90 @@ namespace OpenMS
     // User parameter name for precursor mz error in ppm
     const std::string PRECURSOR_ERROR_PPM_USERPARAM = "OMS:precursor_mz_error_ppm";
 
+    // User parameter name for monoisotopic peak misassignment correction of a precursor (usually an integer x with the correction being x times C13C12_MASSDIFF_U)
```

Comment:
please state in which direction the values should be used (e.g, 1 = subtract (1/charge)*C13C12_MASSDIFF_U from uncorrected precursor_mz